pytorch/Dockerfile

```diff
@@ -100,6 +106,42 @@ RUN wget -q  --no-check-certificate https://raw.githubusercontent.com/oneapi-src
     wget -q  --no-check-certificate https://raw.githubusercontent.com/intel/neural-compressor/master/docker/third-party-programs-pytorch.txt -O /licensing/third-party-programs-pytorch.txt && \
     wget -q  --no-check-certificate https://raw.githubusercontent.com/intel/neural-compressor/master/LICENSE -O /licensing/LICENSE
 
+# Enable passwordless ssh for mpirun
+RUN mkdir /var/run/sshd
+RUN sed -i'' -e's/^#PermitRootLogin prohibit-password$/PermitRootLogin yes/' /etc/ssh/sshd_config \
+        && sed -i'' -e's/^#PasswordAuthentication yes$/PasswordAuthentication yes/' /etc/ssh/sshd_config \
+        && sed -i'' -e's/^#PermitEmptyPasswords no$/PermitEmptyPasswords yes/' /etc/ssh/sshd_config \
+        && sed -i'' -e's/^UsePAM yes/UsePAM no/' /etc/ssh/sshd_config \
+        && echo "Port "$SSHD_PORT"" >> /etc/ssh/sshd_config \
+        && echo "Host *" >> /etc/ssh/ssh_config \
+        && echo "  Port "$SSHD_PORT"" >> /etc/ssh/ssh_config \
+        && echo "  StrictHostKeyChecking no" >> /etc/ssh/ssh_config
+
+EXPOSE ${SSHD_PORT}
+CMD ["/usr/sbin/sshd", "-D"]
+
+RUN useradd -rm -d /home/intel -s /bin/bash intel && \
+    echo 'intel ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers
+USER intel
+WORKDIR /home/intel
+RUN mkdir /home/intel/.ssh
+
+USER root
+RUN chown intel /etc/ssh/ssh_host_ecdsa_key && \
+   chown intel /etc/ssh/ssh_host_rsa_key && \
+   chown intel /etc/ssh/ssh_host_ed25519_key
```

Comment:
Instead of using the already-generated keys, you should generate random keys on startup with this script: https://github.com/intel/transfer-learning/blob/main/docker/hf_k8s/generate_ssh_keys.sh

Reply:
understand. will follow the instructions

Reply:
addressed accordingly